wgsl/index.bs

```diff
@@ -11156,25 +11156,32 @@ In this section, a floating point type may be any of:
 
 Note: Recall that the [=f32=] WGSL type corresponds to the IEEE-754 binary32 format, and the [=f16=] WGSL type corresponds to the IEEE-754 binary16 format.
 
-When converting a floating point scalar value to an [=integer scalar=] type:
-* If the original value is exactly representable in the destination type, then the result is that value.
-* Otherwise, the original value is rounded toward zero.
-    * If the rounded value is exactly representable in the destination type, the result is that value.
-    * Otherwise, the result is the value in the destination type that is closest to the rounded value.
+The <dfn noexport>scalar floating point to integral conversion</dfn> algorithm is:
+<blockquote algorithm="convert a float value to an integral value">
+To convert a floating point scalar value |X| to an [=integer scalar=] type |T|:
+* If the original value |X| is exactly representable in the target type |T|, then the result is that value.
```

Comment:
Since now there is `|X|` naming, is it possible to phrase as "... then the result is |X|" for (a tiny bit more) ease of reading? Just suggestion, assuming mechanically this works.

Reply:
Got it, thanks!

Reply:
I see, in isolation, it should be read as "original value of |X|" than "original value |X|".

Reply:
Adding "of" there is probably an improvement.

Reply:
@dneto0 Thank you so much, it at least helps me a lot to see the phrase as not flat compound but as one with actual head!